src/components/BooksForm.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      category: 'Action',
+    };
+  }
+
+  render() {
+    return (
+      <form>
+        <label htmlFor="title">
+          Title:
+          <input type="text" name="title" />
+        </label>
+        <br />
+        <label htmlFor="category">
+          Category:
+          <select>
+            <option value="Action">Action</option>
+            <option value="Biography">Biography</option>
+            <option value="History">History</option>
+            <option value="Horror">Horror</option>
+            <option value="Kids">Kids</option>
+            <option value="Learning">Learning</option>
+            <option value="Sci-fi">Sci-fi</option>
```

Comment:
This should be from a constant, see requirement [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/CategoryConstants.png)